src/config/passport.ts

```diff
@@ -0,0 +1,24 @@
+import passport from "passport";
+import { Strategy, ExtractJwt } from "passport-jwt";
+import { User } from "../models/user.model";
+
+const options = {
+  jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+  secretOrKey: process.env.JWT!,
+  algorithms: ["HS256"],
+};
+
+const strategy = new Strategy(options, async (payload, done) => {
```

Comment:
I'm not sure that config is a correct place for the strategies. It should be place in auth module